packages/api/src/env.js

```diff
@@ -94,6 +98,20 @@ export function envAll (req, env, ctx) {
     release: env.SENTRY_RELEASE,
     pkg
   })
+
+  // Attach a `Logging` instance, which provides methods for logging and writes
+  // the logs to LogTail. This must be a new instance per request.
+  // Note that we pass `ctx` as the `event` param here, because that's what it
+  // actually is, we seem to have just misnamed it "ctx".
```

Comment:
It's nuanced. The `ctx` is "the object that has the `.waitUntil()` and `.passThroughOnException()` methods on it". 
<img width="455" alt="Screenshot 2022-05-13 at 13 20 43" src="https://user-images.githubusercontent.com/58871/168282003-44415ee9-d0df-4880-847b-c7fb7b631f2f.png">
– https://developers.cloudflare.com/workers/runtime-apis/fetch-event/#parameters

> While an incoming HTTP request is still given the "fetch" name, a Module Worker does not surface the FetchEvent interface. Instead, Module Workers receive the [Request](https://developers.cloudflare.com/workers/runtime-apis/request/) and must reply with a [Response](https://developers.cloudflare.com/workers/runtime-apis/response/) directly.

in practice the worker recieves a `ctx` param that is in fact just the FetchEvent... IT'S BOTH!

Reply:
Ah, this is why I was slightly perplexed by it! I'm updating the comment. Thanks.